### PR TITLE
chore: Add API GetKeyInfoByName. In CreateAccount doc, say to use it to check

### PR DESCRIPTION
**Pull request by jefft0**
As we discussed, add GetKeyInfoByName, GetKeyInfoByAddress and GetKeyInfoByNameOrAddress. Clarify the CreateAccount doc.

### CODE REVIEW

File: service/gnomobiletypes/gnomobiletypes.go [hunk 1]
```diff
@@ -58,6 +58,14 @@ type ListKeyInfoResponse struct {
 	Keys []*KeyInfo `json:"key_info" yaml:"key_info"`
 }
 
+type GetKeyInfoByNameRequest struct {
+	NameOrBech32 string `json:"name_or_bech32" yaml:"name_or_bech32"`
```
Comment (D4ryl00) [inline service/gnomobiletypes/gnomobiletypes.go:62]: It's confusing that the field is `NameOrBech32` and the method is `GetKeyInfoByNameRequest`, so it only by name or it's both name or bech32?

**Review by iuricmp — APPROVED** [review]

File: service/gnomobiletypes/gnomobiletypes.go (hunk 1 above)
Comment (jefft0) [reply service/gnomobiletypes/gnomobiletypes.go:62]: As I replied to your other comment, maybe we should have all three:
GetKeyInfoByName(name), GetKeyInfoByAddress(bech32Addr) and GetKeyInfoByNameOrAddress(nameOrBech32). What do you think?

Comment (D4ryl00) [reply service/gnomobiletypes/gnomobiletypes.go:62]: Yes I agree.

Comment (jefft0) [reply service/gnomobiletypes/gnomobiletypes.go]: Hi @D4ryl00 . I force pushed a commit with these three functions. Can you review please?

**Review by D4ryl00 — APPROVED** [review]
LGTM

**Pull request merged**